GOVERNANCE.md

```diff
@@ -0,0 +1,114 @@
+- Plugins:
+    - https://github.com/fermyon/spin-test
+    - https://github.com/fermyon/cloud-plugin
+    - https://github.com/fermyon/spin-cloud-gpu
+    - https://github.com/fermyon/spin-plugins
+    - https://github.com/fermyon/platform-plugin
+- Triggers:
+    - https://github.com/fermyon/spin-trigger-command
+    - https://github.com/fermyon/spin-trigger-sqs
+    - https://github.com/fermyon/spin-trigger-cron
+    
+
+Each repository is subject to the same overall governance model, but has different teams of people (“maintainers”) with permissions and access to the repository. This is meant to increase diversity of maintainers in the Spin project and also increases the velocity of code changes. Major changes and features to the project (including changes to project governance) are to be proposed through the [Spin Improvement Proposal](docs/content/sips/index.md) process.
+
+## Spin Project Maintainers
+
+Project maintainers are responsible for activities surrounding the development and release cycle of a specific repository they own. Technical decisions for code resides with the project maintainers, unless there is a decision related to cross maintainer groups that cannot be resolved by those groups. Those cases can be escalated to the Spin Governance Committee (discussed below).
+
+Project maintainers do not need to be software developers. No explicit role is placed upon them, and they can be anyone appropriate for the work being produced.
+
+Project maintainers and emeritus maintainers for each project should be outlined in the MAINTAINERS.md file in the corresponding GitHub repository.
+
+Changes to project maintainers use the following:
+
+- A project maintainer may step down by emailing the mailing list. When a project maintainer steps down, they become an emeritus maintainer.
+- Project maintainers MUST remain active on the project. If they are unresponsive for > 3 months, they will lose project maintainer-ship, unless a [supermajority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) (two-thirds majority) of the project maintainers of the given project agrees to extend the period to be greater than 3 months. If two-thirds majority is not possible, the Spin Governance Committee members will be included in the group of people voting on this decision in addition to the project maintainers of the project.
+- New maintainers can be added to a project by a [supermajority](https://en.wikipedia.org/wiki/Supermajority#Two-thirds_vote) vote of the project’s existing maintainers. New maintainers must be nominated by existing maintainers. If quorum is not met, the Spin Govenance Committee members will be included in this vote.
+- A maintainer may be removed for a [code of conduct](../../../CODE_OF_CONDUCT.md) violation via a supermajority vote of the Spin Governance Committee. Code of conduct violations may be submitted to any member(s) on the Spin Governance Committee by email. See email information on MAINTAINERS.md.
```

Comment:
```suggestion
- A maintainer may be removed for a [code of conduct](CODE_OF_CONDUCT.md) violation via a supermajority vote of the Spin Governance Committee. Code of conduct violations may be submitted to any member(s) on the Spin Governance Committee by email. See email information on MAINTAINERS.md.
```

Reply:
updated location of GOVERNANCE.md so that should cover this issue. Thanks!